sui/manage-coins.js

```diff
@@ -0,0 +1,218 @@
+const { TransactionBlock } = require('@mysten/sui.js/transactions');
+const { saveConfig } = require('../evm/utils');
+const { Command } = require('commander');
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet } = require('./sign-utils');
+const { printInfo } = require('../evm/utils');
+const chalk = require('chalk');
+const { loadSuiConfig } = require('./utils');
+
+const SUI_COIN_ID = '0x2::sui::SUI';
+
+class CoinManager {
```

Comment:
using a class here feels a bit overkill since we never instantiate it, or export it. But fine to keep it and see how it goes

Reply:
yeah, it looks like an overkill by using a class. but my purpose is only for grouping coins-management functions together (there're too many functions) without create a separate file for them. it's easier for me to see overall code, especially when fold or expand code in editor.